src/test/scala/io/iohk/ethereum/transactions/PendingTransactionsManagerSpec.scala

```diff
@@ -115,14 +115,17 @@ class PendingTransactionsManagerSpec extends FlatSpec with Matchers with ScalaFu
     pendingTransactionsManager ! AddOrOverrideTransaction(firstTx)
     peerManager.expectMsg(PeerManagerActor.GetPeers)
     peerManager.reply(Peers(Map(peer1 -> Handshaked)))
+    Thread.sleep(Timeouts.normalTimeout.toMillis)
```

Comment:
It is better to have an expectation with a timeout than a `Thread.sleep`. Looking at the code I don't believe the waiting between the individual messages is required. My guess is it's the total waiting time since the first one that did the trick, so I would prefer a `longTimeout` in the expectation.

However, given how crazy this has been, I don't want to spend too much time on this, especially as part of this task...